ozaria/site/common/ozariaUtils.js

```diff
@@ -30,18 +28,31 @@ export const findNextLevelsBySession = (sessions, levels, levelStatusMap) => {
     levelDataMap = levels
   }
   for (const [levelOriginal, level] of Object.entries(levelDataMap)) {
-    if (levelStatusMap[levelOriginal] === 'started') {
+    const levelStatus = levelStatusMap[levelOriginal]
+    const isLevelStarted = typeof levelStatus === 'string' && levelStatus === 'started'
+    const isLevelCompleted = (typeof levelStatus === 'string' && levelStatus === 'complete') || (typeof levelStatus === 'number' && levelStatus > 0)
```

Comment:
Is levelStatus > 0 a completed level?

Reply:
For capstone level played in stages, levelStatus = last completed stage / 'started', if no stage has been completed. Hence levelStatus > 0 would mean that some stages of the capstone have been completed and we need to look at the levels unlocked by those stages.

Reply:
That makes sense. I think the naming confused me. Instead of `isLevelCompleted`, is it more accurately `hasUnlockedLevels`?

Reply:
I can split this statement into 2, one for normal levels `isLevelCompleted` (i.e. when levelStatus = complete), and other for capstone played in stages `hasCompletedStages`. Does that make sense?